test/source/test.ts

```diff
@@ -106,7 +107,11 @@ const startMockApiAndCopyBuild = async (t: AvaContext) => {
   const address = mockApi.server.address();
   if (typeof address === 'object' && address) {
     const result = await asyncExec(`sh ./scripts/config-mock-build.sh ${buildDir} ${address.port}`);
+    const filepath = `${buildDir}/js/common/core/const.js`;
 
+    const fileContent = await readFile(filepath, { encoding: 'utf-8' });
+    const updatedCode = fileContent.replace(/const (MOCK_PORT) = [^;]+;/g, `const $1 = '${address.port}';`);
+    await writeFile(filepath, updatedCode);
```

Comment:
probably we can perform `MOCK_PORT` replacement in `config-mock-build.sh` script, like we do for `TEST_REPLACEABLE_MOCK_PORT`?

Reply:
Let me check